### DISCUSSION TITLE
Support JWT auth

### DISCUSSION THREAD

**Discussion by kdhrubo** [9tigerio/db2rest#650] [Ideas]
A servlet filter that is capable of validating different aspects of the JWT token + JWKS

https://connect2id.com/products/nimbus-jose-jwt/examples/validating-jwt-access-tokens

**Comment by rajivmca2004** [comment]
I am working on this.

**Comment by kdhrubo** [comment]
Feature to move to enterprise edition

**Comment by kdhrubo** [comment]
@lserveriiev has completed this issue 

https://github.com/9tigerio/db2rest/issues/713

Hence closing.
